parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/DMLStatement.g4

```diff
@@ -595,7 +595,7 @@ inlineAnalyticView
     ;
 
 whereClause
-    : WHERE expr
+    : WHERE expr pivotInClause?
```

Comment:
Why add pivotInClause in where sytax?

Reply:
Sir it process 'IN' condition. Sql case id explain_for_select_where_colon uses it
Since I can see in documentation that it appears after Where clause, I have added it there only. 
There are 3 ways to process the grammar. Adding it in either of these would enable parsing.
-Select Queries 
-Where clause
-Expr
If where clause is not the appropriate place, I can consider adding it in other two options
Reference:
https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/IN-Condition.html

Reply:
The IN condition may already be defined in the predicate sytax. Can you take a look?

Reply:
yes sir